uber/templates/preregistration/index.html

```diff
@@ -121,9 +121,11 @@
         <a href="form"><button class="btn btn-default">Add Another Person</button></a>
     </div>
 </div>
+
 </div>
+{% include "preregistration/disclaimers.html" %}
+
 </div>
 
-{% include "preregistration/disclaimers.html" %}
 </div>
```

Comment:
I had to move disclaimers.html above the DIV in order to get it to display correctly on the page.  Someone with more bootstrap-y knowledge should review this stuff and make sure I didn't mess it up too bad.